FogCreek/Glitch-Community
# Superb guide: Faster ssr compilation, better server timeouts

## Links
https://superb-guide.glitch.me

## Changes:
- Added timeouts to all outbound requests (node defaults to no timeouts so dropped requests sat around forever)
- Reduced the api timeout from 5s to 1s because if it takes more than a second it's probably dropped
- Don't polyfill client code for when transpiling for ssr
- Log at the start and end of ssr builds, with a perf measurement

## How To Test:
Join the project, `refresh`, and see how long it takes to load a page. It should average about 45s to transpile, from 75s before. That's still slower than I'd like, but it is an improvement.
